### ISSUE TITLE
How to Suppress XSS notifications?

### ISSUE THREAD

**Issue by hipunk** [hackademix/noscript#223]
Hi,

in the FAQ https://noscript.net/faq#qa4_3 it says:

> Q:   Can I turn off Anti-XSS activity notifications?
> A:   Yes, you can, just toggle the Noscript Options|Notifications|XSS preference. Of course you will still able to monitor NoScript Anti-XSS activity log in the Browser Console (Firefox) or Error Console (SeaMonkey), and you will get an extra "XSS" menu inside the NoScript contextual menu whenever an XSS attempt is detected, featuring all the actions usually accessed from the notification bar. 

Where is the Notifications pane? I'm on Firefox 96.0.3 (64-bit) NoScript Version 11.2.19, Linux, macOS and Windows. Either the FAQ is wrong or I am being served weird binaries.

What I want to achieve: I want every XSS positive (even the false ones) to be blocked by default. If the website breaks because of that, it shall break. No popups.

Greetings

**Comment by hackademix** [comment]
Yes, unfortunately the FAQ is still outdated, referring to NoScript "Classic" (version 5.x and below).
Currently there's no way to turn off just the notifications.
I'm putting this in my TODO list, thanks.
